### ISSUE TITLE
org.gradle.api.GradleException: One or more exceptions occurred during analysis

### ISSUE THREAD

**Issue by sivaswami** [jeremylong/dependency-check-gradle#56]
Hi,
I noticed the below stack trace and Gradle task dependencyCheckAnalyze fails if any violations are found.
any inputs?

```

Caused by: org.gradle.api.GradleException: One or more exceptions occurred during analysis
        at sun.reflect.GeneratedConstructorAccessor609.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
        at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:255)
        at org.owasp.dependencycheck.gradle.tasks.Check.check(Check.groovy:119)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.doExecute(DefaultTaskClassInfoStore.java:141)
        at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:134)
        at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:123)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:692)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:675)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.execute(ExecuteActionsTaskExecuter.java:115)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.execute(ExecuteActionsTaskExecuter.java:109)
        at org.gradle.internal.Transformers$4.transform(Transformers.java:169)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:106)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:56)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:109)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:90)
        ... 23 more
Caused by: org.owasp.dependencycheck.exception.ExceptionCollection: One or more exceptions occurred during dependency-check analysis
        org.owasp.dependencycheck.xml.pom.PomParseException: Unable to parse pom '/tmp/dctemp79c5a3f9-db86-45bd-8f02-61a9cffc437c/check1305227328897147656tmp/366/pom.xml'
        org.owasp.dependencycheck.xml.pom.PomParseException: Unable to parse pom '/tmp/dctempd2a6a250-0ace-4864-b658-b542b68ec02a/pom8183594392549237733.xml'
        at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:541)
        at org.owasp.dependencycheck.Engine$analyzeDependencies$0.call(Unknown Source)
        at org.owasp.dependencycheck.gradle.tasks.Check.check(Check.groovy:79)
```

**Comment by jeremylong** [comment]
I believe this is related to things already fixed in dependency-check-core - specifically, there were some internal threading issues in the earlier versions of dependency-check and issues if two instances were executing at the same time using the same database - these have all been fixed in 3.0.0 which I am in the process of releasing now.

**Comment by jeremylong** [comment]
@sivaswami if you try 3.0.2 has this issue been resolved?
